src/components/edit-on-github-link/index.tsx

```diff
@@ -0,0 +1,22 @@
+import InlineSvg from '@hashicorp/react-inline-svg'
+import svgGithub from '@hashicorp/flight-icons/svg/github-16.svg?include'
+import s from './style.module.css'
+
+function EditOnGithubLink({
```

Comment:
Split out this component, as I thought it might be useful to test placement options depending how prominent we want this link to be. I think previously we put it at the bottom of the page in our designs simply because we didn't have a nice sidebar to put it in.